src/components/app-top-bar.js

```diff
@@ -460,7 +472,18 @@ const AppTopBar = ({ user, tabIndex, onChangeTab, userManager }) => {
                         })}
                     </Tabs>
                 )}
+                {studyUuid && (
+                    <Box sx={styles.divRunButton}>
```

Comment:
can you rename 'divRunButton' to 'runButtonContainerBox' for example (try to not use html tags names inside styles names)

Reply:
Done